src/components/upload/utils.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Check if the provided file type should be accepted by the input with accept attribute.
+ * https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input#attr-accept
+ *
+ * Inspired by https://github.com/enyo/dropzone
+ *
+ * @param file {File} https://developer.mozilla.org/en-US/docs/Web/API/File
+ * @param acceptedFiles {string}
+ * @returns {boolean}
+ */
+
+export function attrAccept(file, acceptedFiles) {
```

Comment:
should wrap function in directive

Reply:
I wonder that is it necessary to make 'attrAccept' a function because it has been implemented in HTML5.But Ant Design does it.